tests/user.test.js

```diff
@@ -39,7 +40,15 @@ test('Should create a new user', async () => {
     }
   });
 
+  // Each user should also have a character generated for them
+  const character = await prisma.character.findUnique({
+    where: {
+      id: parseInt(response.createUser.user.id)
```

Comment:
Where does `response` appear above this line?

Reply:
It's a few lines above here, where there is a request being made using the createUser mutation

Reply:
Ok I see, if I view the source file directly it shows that lines were collapsed in the view for compactness. My eye skipped that...